app-modules/contact/tests/Organization/EditOrganizationTest.php

```diff
@@ -86,10 +86,20 @@
 
     $request = collect(EditOrganizationRequestFactory::new()->create());
 
+    $domains = [
+        [
+            'domain' => 'acme.com',
+        ],
+        [
+            'domain' => 'purdy.com',
+        ],
```

Comment:
Same question as above

Reply:
Hi @Orrison 
With flatten it's not possible as the repeater is only accepting key-value pair array.